arkane/statmech.py

```diff
@@ -428,6 +427,36 @@ def load(self, pdep=False, plot=False):
         logging.debug('    Reading optimized geometry...')
         coordinates, number, mass = geom_log.load_geometry()
 
+        if self.species.conformer is not None and len(self.species.conformer.modes):
```

Comment:
Why is this commit necessary?  Why are these modes missing?

Reply:
I couldn't figure out how to parse the moments of inertia out of a TeraChem log file (I think they might not be reported, although they are certainly calculated and used internally). TeraChem's manual did not mention anything about it, and I couldn't find it by Googling either. Duminda reported the same thing for Orca, so this functionality could be used when parsing Orca as well.

Reply:
Okay, so normally Arkane builds these modes, but when that data isn't there they just don't get added to the Conformer object?

Reply:
Correct, Arkane will still parse these modes as usual if it can, otherwise we append them here